test_nei.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
```

Comment:
I commited that file by mistake, it's just the one I'm using to see if this is actually working. I will remove it in the next commit as I don't think it's relevant as a test. It just takes some hours and produce plots.